zio-cache/shared/src/main/scala/zio/cache/ScopedCache.scala

```diff
@@ -186,6 +192,24 @@ object ScopedCache {
             }
           }
 
+        def freeExpired: UIO[Int] = ZIO.suspendSucceed {
+          var expiredKey = List.empty[Key]
+          map.entrySet().forEach { entry =>
```

Comment:
I would write it:
```
map.collect { case value: MapValue.Complete if hasExpired(value.ttl) => value }
```
but maybe there's something I miss

Reply:
Here it's a java.util.Map therefore I don't have the collect method